content/resources/chapter-4/address.tsx

```diff
@@ -0,0 +1,220 @@
+publicKeyHash = ripemdHash.hexdigest()
+print(publicKeyHash)`,
+    `from hashlib import sha256wooooo
+
+# Create a program that finds a sha256 hash starting with 5 zeroes.
+# To submit your answer, print it to the terminal using print().
+
+# Type your code here
+`,
+  ],
+  validate: async (answer) => {
+    return [true, undefined]
+  },
+  constraints: [],
+}
+
+const configOne: EditorConfig = {
+  defaultLanguage: 'javascript',
+  languages: {
+    javascript,
+    python,
+  },
+}
+
+const configTwo: EditorConfig = {
+  defaultLanguage: 'javascript',
+  languages: {
+    javascript,
+    python,
+  },
+}
+
+export default function AddressResources({ lang }) {
+  const t = useTranslations(lang)
+
+  const [codeOne, setCodeOne] = useState(
+    configOne.languages[configOne.defaultLanguage].defaultCode?.[0]
+  )
+  const [codeTwo, setCodeTwo] = useState(
+    configTwo.languages[configTwo.defaultLanguage].defaultCode?.[1]
+  )
+  const [languageOne, setLanguageOne] = useState(configOne.defaultLanguage)
+  const [languageTwo, setLanguageTwo] = useState(configTwo.defaultLanguage)
+  const [challengeOneIsToggled, setChallengeOneIsToggled] = useState(false)
+  const [challengeTwoIsToggled, setChallengeTwoIsToggled] = useState(false)
+
+  const challengeOneToggleSwitch = () => {
+    setChallengeOneIsToggled(!challengeOneIsToggled)
+  }
+
+  const challengeTwoToggleSwitch = () => {
+    setChallengeTwoIsToggled(!challengeTwoIsToggled)
+  }
+
+  const handleSetLanguageOne = (value) => {
+    setLanguageOne(value)
+    setCodeOne(configOne.languages[value].defaultCode?.[0])
+  }
+
+  const handleSetLanguageTwo = (value) => {
+    setLanguageTwo(value)
+    setCodeTwo(configOne.languages[value].defaultCode?.[1])
+  }
+
+  const handleBeforeMount = (monaco) => {
+    monaco.editor.defineTheme('satoshi', {
+      base: readOnlyOptions.theme,
+      inherit: true,
+      rules: [],
+      colors: {
+        'editor.background': '#00000000',
+        'editor.lineHighlightBorder': '#00000000', // 4th channel is for transparency
+      },
+    })
+  }
+
+  const handleMount = (_editor, monaco) => {
+    monaco.editor.setTheme('satoshi')
+  }
+
+  return (
+    <ResourcePage
+      lang={lang}
+      readingResources={
+        <>
+          <Text className="mt-[25px] text-xl font-bold">
+            {t('chapter_four.resources.address.hash_algo_heading')}
+          </Text>
+          <Text>{t('chapter_four.resources.address.hash_algo_paragraph')}</Text>
+          <Text className="mt-[25px] text-xl font-bold">
+            {t('chapter_four.resources.address.wpkh_heading')}
+          </Text>
+          <Text>{t('chapter_four.resources.address.wpkh_paragraph')}</Text>
+          <Text className="mt-[25px] text-xl font-bold">
+            {t('chapter_four.resources.address.network_heading')}
+          </Text>
+          <Text>{t('chapter_four.resources.address.network_paragraph')}</Text>
+        </>
+      }
+      tipsResources={'hello world'}
```

Comment:
Is this intentional?
<img width="103" alt="image" src="https://github.com/saving-satoshi/saving-satoshi/assets/3502409/1b5fd4f6-0cf0-407c-8b53-5576450295ef">

Reply:
nope!